packages/quantic/force-app/main/default/lwc/quanticResultQuickview/quanticResultQuickview.js

```diff
@@ -280,6 +289,10 @@ export default class QuanticResultQuickview extends LightningElement {
     }
   }
 
+  onIframeLoaded() {
```

Comment:
nitpick: most examples I found online capitalize `iframe` like two words (Inline Frame -> `IFrame` in pascal case / `iFrame` in camel case).